### PR TITLE
[Feature] Light/Dark Mode Risk Legend

### PR DESCRIPTION
**Pull request by eisvogel**
This is the first PR in the series of updating our color scheme to the latest design specifications.

Starting with this PR the new colors have been introduced, eventually replacing the old colors set all together. To avoid merge conflicts whilst editing Storyboards in multiple branches it was agreed on to merge these changes incrementally.

In this PR the `RiskLegendViewController` was updated to the latest color scheme and check for light and dark mode support

### CODE REVIEW

**Comment by i336616** [comment]
Minor comment, should the scrollview background be Color.systemBackground?

IMG_1994

**Comment by i336616** [comment]
Image backgrounds in App Informationen should be clear
IMG_1995
IMG_1997
IMG_1998
IMG_2001

**Comment by EisvogelSAP** [comment]
This PR focus on Risk Legend. ;-) `ExposureDetectionViewController` will be coming soon.

**Comment by AdnanZiSAP** [comment]
> Minor comment, should the scrollview background be Color.systemBackground?
> 
> IMG_1994

It should but I think a general bug should be reported for this. Shouldn't ever be blue. 

Also, further down the screen there's some lorem ipsum text.

Will you report this to Elmar or should I do it? :) Any way is fine to me.

**Comment by AdnanZiSAP** [comment]
> Image backgrounds in App Informationen should be clear
> IMG_1995

Just for the sake or documenting it, a lot of screens have the issue like this, where the illustration is not transparent.

File: src/xcode/ENA/ENA.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -132,6 +132,7 @@
 		71B8044F248526B600D53506 /* DynamicTableViewSpaceCell.swift in Sources */ = {isa = PBXBuildFile; fileRef = 71B8044E248526B600D53506 /* DynamicTableViewSpaceCell.swift */; };
 		71B804522485272F00D53506 /* RiskLegendNumberedTitleCell.swift in Sources */ = {isa = PBXBuildFile; fileRef = 71B804512485272F00D53506 /* RiskLegendNumberedTitleCell.swift */; };
 		71B804542485273C00D53506 /* RiskLegendDotBodyCell.swift in Sources */ = {isa = PBXBuildFile; fileRef = 71B804532485273C00D53506 /* RiskLegendDotBodyCell.swift */; };
+		71F2E57B2487AEFC00694F1A /* ena-colors.xcassets in Resources */ = {isa = PBXBuildFile; fileRef = 71F2E57A2487AEFC00694F1A /* ena-colors.xcassets */; };
```
Comment (aleksandr-tikhonov) [inline src/xcode/ENA/ENA.xcodeproj/project.pbxproj:135]: Can we please  have only one .asset file for colors ? that would be single source of truth

**Review by aleksandr-tikhonov — CHANGES_REQUESTED** [review]

**Review by i336616 — APPROVED** [review]
👍

**Comment by AdnanZiSAP** [comment]
Also for the sake of documenting it:

IMG_47578BC4B487-1

Sections backgrounds.

File: src/xcode/ENA/ENA.xcodeproj/project.pbxproj (hunk 1 above)
Comment (EisvogelSAP) [reply src/xcode/ENA/ENA.xcodeproj/project.pbxproj:135]: Sure, again. This is the first in a series of PRs. I will incrementally update all screen to the latest color scheme. Once done, the old scheme will be removed.

This way it is easier to keep track on things. This is agreed upon with Hao.

**Comment by EisvogelSAP** [comment]
Thanks guys for the comments. Please bear in mind this PR only updates a single VC. All other screens will follow one by one to avoid large merge conflicts while I update them and you work on other things. This is agreed upon with Hao.

Sorry for the confusion.

**Comment by AdnanZiSAP** [comment]
Perhaps until the dark mode looks at least ok on all screens, we should prevent the app of going into the dark mode. What do you guys think?

**Comment by aleksandr-tikhonov** [comment]
> Perhaps until the dark mode looks at least ok on all screens, we should prevent the app of going into the dark mode. What do you guys think?

I also thought about it, I think, it's good idea

**Comment by EisvogelSAP** [comment]
We should have disabled dark mode from the very beginning. Why should we start worrying about that now? :-D

**Review by aleksandr-tikhonov — APPROVED** [review]

**Comment by AdnanZiSAP** [comment]
> > Minor comment, should the scrollview background be Color.systemBackground?
> 
> It should but I think it's a general bug should be reported for this. Shouldn't ever be blue.
> 
> Also, further down the screen there's some lorem ipsum text.
> 
> Will you report this to Elmar or should I do it? :) Any way is fine to me.

Reported to Elmar.

**Comment by AdnanZiSAP** [comment]
> We should have disabled dark mode from the very beginning. Why should we start worrying about that now? :-D

Because we're open source now. And this is the first time I saw how it looks. 😅

**Pull request merged**